src/Stripe/Infrastructure/Requestor.cs

```diff
@@ -29,6 +31,57 @@ public static string PostString(string url, StripeRequestOptions requestOptions)
             return ExecuteRequest(wr);
         }
 
```

Comment:
I had posted from another issue (I believe you commented on), that looked like an easier (less code way) to do the file upload. I'm not sure if it would work the same, but it would be worth investigating to see if we can reduce some code.